README.md

```diff
@@ -1,6 +1,6 @@
 # Clima ☁
 
-Beautiful, minimal, and fast weather app. (Requires Android 6.0 or later)
```

Comment:
Why remove this line?

Reply:
It's really not necessary

Reply:
We can discuss this later, but it's not related to this PR. Please do it in another one.